extensions/data-transfer/portability-data-transfer-koofr/src/main/java/org/datatransferproject/transfer/koofr/common/KoofrClient.java

```diff
@@ -102,13 +103,9 @@ public boolean fileExists(String path) throws IOException, InvalidTokenException
       if (code == 404) {
         return false;
       }
-      throw new IOException(
-          "Got error code: "
-              + code
-              + " message: "
-              + response.message()
-              + " body: "
-              + response.body().string());
+      ResponseBody body = response.body();
```

Comment:
since you're introducing Koofr exception anyway, and koovr client depends on okhttp3, consider just passing `ResponseBody` in as the third param, and let the KoofrClietnIOException do all these work (and then you can delete the other places this is copy/pasted)

Reply:
Switched to using okhttp Response for creating KoofrClientIOException